cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/mod.rs

```diff
@@ -43,6 +44,16 @@ pub(crate) fn asset_hub_rococo_location() -> Location {
 	)
 }
 
+pub(crate) fn asset_hub_westend_location() -> Location {
+	Location::new(
+		2,
+		[
+			GlobalConsensus(ByGenesis(WESTEND_GENESIS_HASH)),
+			Parachain(AssetHubWestend::para_id().into()),
+		],
+	)
+}
```

Comment:
this is defined in `bridge-hub-westend` files, so I am expecting a BHW-relative location for a fn called `ahw_location()`

That relative location is:

```suggestion
pub(crate) fn asset_hub_westend_location() -> Location {
	Location::new(1, [Parachain(AssetHubWestend::para_id().into())])
}
```

If you need it relative to something else (Ethereum maybe), at least rename the fn name accordingly.

Reply:
Addressed in https://github.com/paritytech/polkadot-sdk/pull/7956/commits/9e5ed82a9e2a3cffe012a6307e6bc0043560ca20.